README.rst

```diff
@@ -53,31 +34,16 @@ Mac/Linux
 
 .. code-block:: console
 
-    pip install virtualenv
-    virtualenv <your-env>
+    python3 -m venv <your-env>
     source <your-env>/bin/activate
-    <your-env>/bin/pip install google-cloud-trace
+    <your-env>/bin/pip install /path/to/library
 
 
 Windows
 ^^^^^^^
 
 .. code-block:: console
 
-    pip install virtualenv
-    virtualenv <your-env>
+    python3 -m venv <your-env>
     <your-env>\Scripts\activate
-    <your-env>\Scripts\pip.exe install google-cloud-trace
-
-Next Steps
-~~~~~~~~~~
-
--  Read the `Client Library Documentation`_ for Cloud Trace API
-   to see other available methods on the client.
--  Read the `Cloud Trace Product documentation`_ to learn
-   more about the product and see How-to Guides.
--  View this `README`_ to see the full list of Cloud
-   APIs that we cover.
-
-.. _Cloud Trace Product documentation:  https://cloud.google.com/trace/docs
-.. _README: https://github.com/googleapis/google-cloud-python/blob/main/README.rst
+    <your-env>\Scripts\pip.exe install \path\to\library
```

Comment:
None of the `README.rst` changes look correct to me:  why is this file being templated without proper links?